mozci/console/commands/push.py

```diff
@@ -211,6 +223,38 @@ def handle(self) -> None:
                     f"Classification and regressions details for push {push.push_uuid} were saved in {filename} JSON file"
                 )
 
+            # Send a notification when some emails are declared in the config
+            emails = config.get("emails")
```

Comment:
We should use two different configs to store the lists of emails (the receivers of one won't be the same as the receivers of the other).